packages/block-editor/src/components/line-height-control/index.js

```diff
@@ -63,8 +63,8 @@ export default function LineHeightControl( {
 		const wasTypedOrPasted = [ 'insertText', 'insertFromPaste' ].includes(
 			action.payload.event.nativeEvent?.inputType
 		);
-		state.value = adjustNextValue( state.value, wasTypedOrPasted );
-		return state;
+		const value = adjustNextValue( state.value, wasTypedOrPasted );
+		return { ...state, value };
```

Comment:
This part didn't have to change. It's just proof that now the state does not [have to be mutated in order to work](https://github.com/WordPress/gutenberg/pull/37164/files#r801084302).